content/master/get-started/get-started-with-operations.md

```diff
@@ -0,0 +1,367 @@
+            port = 443
+
+            # Get SSL certificate info
+            context = ssl.create_default_context()
+            with socket.create_connection((hostname, port)) as sock:
+                with context.wrap_socket(sock, server_hostname=hostname) as ssock:
+                    cert = ssock.getpeercert()
+
+            # Parse expiration date
+            expiry_date = datetime.strptime(cert['notAfter'], '%b %d %H:%M:%S %Y %Z')
+            days_until_expiry = (expiry_date - datetime.now()).days
+
+            # Return results in operation output
+            response.set_output(rsp, {
+                "hostname": hostname,
+                "certificateExpires": cert['notAfter'],
+                "daysUntilExpiry": days_until_expiry,
+                "status": "warning" if days_until_expiry < 30 else "ok"
+            })
+```
+
+<!-- vale Crossplane.Spelling = NO -->
+**The Operation runs once to completion, like a Kubernetes Job.**
+<!-- vale Crossplane.Spelling = YES -->
+
+When you create the Operation, Crossplane runs the function pipeline. The
+function checks SSL certificate expiry for google.com and returns the results
+in the operation's output.
+
+This basic example shows the concept. In the walkthrough below, you create
+a more realistic Operation that reads Kubernetes Ingress resources and
+annotates them with certificate expiry information for monitoring tools.
+
+## Prerequisites
+
+This guide requires:
+
+* A Kubernetes cluster with at least 2 GB of RAM
+* The Crossplane v2 preview [installed on the Kubernetes cluster]({{<ref "install">}}) with Operations enabled
+
+{{<hint "tip">}}
+Enable Operations by adding `--enable-operations` to Crossplane's startup
+arguments. If using Helm:
+
+```shell
+helm upgrade --install crossplane crossplane-stable/crossplane \
+  --namespace crossplane-system \
+  --set args='{"--enable-operations"}'
+```
+{{</hint>}}
+
+## Create an operation
+
+Follow these steps to create your first Operation:
+
+1. [Create a sample Ingress](#create-a-sample-ingress) for certificate checking
+1. [Install the function](#install-the-function) you want to use for the
+   operation
+1. [Create the Operation](#create-the-operation) that checks the Ingress
+1. [Check the Operation](#check-the-operation) as it runs
+
+### Create a sample Ingress
+
+Create an Ingress that references a real hostname but doesn't route actual
+traffic:
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: example-app
+  namespace: default
+spec:
+  rules:
+  - host: google.com
+    http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: nonexistent-service
+            port:
+              number: 80
+```
+
+Save as `ingress.yaml` and apply it:
+
+```shell
+kubectl apply -f ingress.yaml
+```
+
+### Grant Ingress permissions
+
+Operations need permission to access and change Ingresses. Create a ClusterRole
```

Comment:
(this comment applies in many places in this PR)

in other guides, we've used code blocks for type names, do you want to continue using that pattern here too? e.g.
```suggestion
`Operations` need permission to access and change `Ingresses`. Create a `ClusterRole`
```

that seems to be the existing pattern on the other guides:
* https://deploy-preview-956--crossplane.netlify.app/master/get-started/get-started-with-operations/
* https://deploy-preview-956--crossplane.netlify.app/master/get-started/get-started-with-managed-resources/

Reply:
We're pretty inconsistent with this in general across the docs. I used to like it but now I'm less of a fan (I feel like it makes the docs a bit harder to read). That said good point that the getting started guides at least are consistent, I should match them.